config/locales/cy.yml

```diff
@@ -202,7 +202,7 @@ cy:
       assistance_question: A gawsoch chi unrhyw gymorth i ddefnyddio'r gwasanaeth hwn heddiw?
       before_you_start: Cyn i chi ddechrau
       dissatisfied: Anfodlon
-      e_vehicle_info: Mae gwybodaeth ar gael am gerbydau trydan.
+      e_vehicle_info:
```

Comment:
Did you add the new text to the Translations log (https://docs.google.com/spreadsheets/d/1Wips8w48mUoLE3UqY-TIeoHJcW9l371qUuRyCqbl5J0/edit#gid=0) ? We should capture that there's a need for new text whenever we lose a translation or create new strings so we can get the new wording when we next pay a translator.

Reply:
> Looks good, nice work Jon :)

Great thanks Martin 👍

Reply:
> Did you add the new text to the Translations log (https://docs.google.com/spreadsheets/d/1Wips8w48mUoLE3UqY-TIeoHJcW9l371qUuRyCqbl5J0/edit#gid=0) ? We should capture that there's a need for new text whenever we lose a translation or create new strings so we can get the new wording when we next pay a translator.

Thanks Max, good to know, i've now updated the doc.